main.go

```diff
@@ -257,3 +258,17 @@ func setOptionDefaults(opt *ctrl.Options) {
 		opt.LeaderElectionID = "c1f6bfd2.kueue.x-k8s.io"
 	}
 }
+
+func setInternalCertManagementDefaults(certOpts *configv1alpha2.InternalCertManagement) {
+	if *certOpts.Enable {
+		if certOpts.ServiceName == "" {
+			certOpts.ServiceName = "kueue-webhook-service"
```

Comment:
Let's make  default `ServiceName`... a block of constants.

Reply:
It makes sense.

Reply:
Done.